apps/coordinator/src/components/ScriptExplorer/DirectSignatureImporter.jsx

```diff
@@ -63,9 +57,6 @@ class DirectSignatureImporter extends React.Component {
     return SignMultisigTransaction({
       network,
       keystore,
-      inputs,
-      outputs,
-      bip32Paths,
```

Comment:
these are no longer needed if the psbt is being passed in (which it is on line 72)